src/components/Icons/AddressBook.vue

```diff
@@ -31,6 +32,10 @@ export default {
       validator: function (value) {
         return ['xxl', 'xl', 'l', 'm', 's'].includes(value);
       }
+    },
+    title: {
+      type: String,
+      default: 'Address Book Icon'
```

Comment:
1. when would we use a different title?
2. does this show on screen? if so, can we translate?